src/ntapfuse.c

```diff
@@ -68,56 +68,71 @@ struct fuse_operations ntapfuse_ops = {
 void
 usage ()
 {
-  printf ("ntapfuse mount <basedir> <mountpoint>\n");
-  
-  exit (0);
+    printf ("ntapfuse mount <basedir> <mountpoint>\n");
+    
+    exit (0);
 }
 
 int
 main (int argc, char *argv[])
 {
-  if (argc < 3)
-    usage ();
+    if (argc < 3){
```

Comment:
nitpick: add a space before {
Also, audit other areas. 
(For consistency in code, we are very strict at NetApp with spaces, newlines, indentation, etc., just FYI - I hope this is helping you learn. These minor tweaks help make code more readable, especially when you have a big codebase and a lot of people making changes to the code)

Reply:
Thanks for all the advices, I didn't pay attention to this kind of details at all. Will keep it in mind